apps/dashboard/src/components/engine/contract-subscription/contract-subscriptions-table.tsx

```diff
@@ -93,9 +93,9 @@ export const ContractSubscriptionTable: React.FC<
         const explorer = chain?.explorers?.[0];
         if (!explorer) {
           return (
-            <AddressCopyButton
+            <CopyAddressButton
```

Comment:
use the new `WalletAddress` component instead that shows social profiles

Reply:
@MananTank this is for showing contract address btw, do we want to use the "social" feature on them?

I will update the code for u to review anyway

Reply:
Ahh may bad. Thought this was wallet addess